src/net/NetworkClient.h

```diff
@@ -5,64 +5,71 @@
 #include <boost/asio/ssl.hpp>
 #include "util/Datagram.h"
 
```

Comment:
Addressed with a comment (it also explains a couple more details on how to use NetworkClient).